validator-auction-rest-server/lib/EthEventsClient.js

```diff
@@ -193,6 +200,22 @@ export default class EthEventsClient {
         return startPriceWEI.mul(_1.add(t)).div(_1.add(t).add(decay))
     }
 
+    static calculateAllSlotPrices(startInSeconds, durationInDays, startPriceWEI, ticks = 312) {
```

Comment:
We do you do this on the server? I think this can be done on the client.

Reply:
That would mean to have business logic in the client and also to use another library (BN.js) in the client. Also you would end with having duplicate code (price function) in the client and server.

Reply:
I think BN should only be on the client and the price function should also only be on the client. I would not duplicate that business logic but only having it on the client. There is no need doing it on the server. We can discuss this after the daily.

Reply:
Team decided this is necessary to change at the moment.